src/utils/progressGraph.js

```diff
@@ -122,6 +122,32 @@ export const progressGraphOptions = {
         },
     },
     labels: ['Average Results'],
+    methods: {
+        update: function () {
+            console.log(this)
```

Comment:
I forget to remove these all the time and get called out for it, start to get in the habit of removing them.